source/by-role/system-admin/configuration.rst

```diff
@@ -885,6 +892,34 @@ Here are the environment variables used for configuring Studio's access tokens f
 
 See :ref:`studio-access-tokens` for more information.
 
+"""""""""""""""""""""""""""""""
```

Comment:
This should be in the `developer` section _and_ here. Turn into an include or similar to have it in both.